io.emeric.watchflower.yml

```diff
@@ -0,0 +1,98 @@
+app-id: io.emeric.watchflower
+runtime: org.kde.Platform
+runtime-version: '5.15'
```

Comment:
```suggestion
runtime-version: '5.15-21.08'
```

Reply:
Can I ask why? I just need Qt 5.15, the version of the KDE SDK doesn't matter at all for this application.

Reply:
this is just the versiobn based of the latest freedesktop-sdk. Policy: use the latest.